internal/pkg/comm/server.go

```diff
@@ -234,6 +237,7 @@ func (gServer *GRPCServer) Stop() {
 func (gServer *GRPCServer) AppendClientRootCAs(clientRoots [][]byte) error {
```

Comment:
This method (AppendClientRootCAs) is actually not used in production code. 
Let's get rid of it shall we?